KickassTooltips.sublime-settings

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I suggest merging all settings into one file `SublimeKickAssemblerC64/KickAssembler (C64).sublime-settings`. This means that for the user only one package exist, not 2. Good or bad idea?

Reply:
Not a good idea.